components/airtable/actions/create-single-record/create-single-record.js

```diff
@@ -8,15 +8,41 @@ module.exports = {
   type: "action",
```

Comment:
Make sure you bump the version of this component.

```suggestion
  version: "0.0.12",
  type: "action",
```

Reply:
(Github does not allow me to comment on a line that's far from a changed section, which is the case of line no. 7 here)